# Pythons M2Crypto raises exception in ssl_ctx_load_verify_locations when loading certificates
ctx.load_verify_locations(str(m.ca_file))
